docs/product/security/tax-documents.mdx

```diff
@@ -6,10 +6,11 @@ description: "Access Sentry's tax documents."
 
 Below you'll find Sentry's 2023 tax documents, including the tax residency certificate, W9, 10F, No PE undertaking, and DGT-1 forms. If the document you're looking for isn't listed here, visit [help.sentry.io](http://help.sentry.io/).
 
-- [Tax Residency Certificate CY2023 (General Version)](/pdfs/2023-Form-6166_general.pdf)
 - [W9 CY2024](/pdfs/W-9-2024-Functional-Software-Inc-signed.pdf)
-- [India: No PE Undertaking CY 2023](/pdfs/Form_10F_Filed_Form_EXTERNAL_04-2023_to_12-2023.pdf)
+- [Tax Residency Certificate CY2023 (General Version)](/pdfs/2023-Form-6166_general.pdf)
+- [Tax Residency Certificate CY2024 (General Version)](/pdfs/Tax_residency_certificate_CY2024.pdf)
+- [India: No PE Undertaking CY 2023](/pdfs//No-PE-Undertaking-Form_2023_Signed.pdf)
 - [India: No PE Undertaking CY 2024](/pdfs/India-No-PE-Undertaking-Form_2024_Signed.pdf)
-- [No PE Undertaking CY 2023](/pdfs//No-PE-Undertaking-Form_2023_Signed.pdf)
+- [India: Electronic _10F_04-2023_to_12-2023](/pdfs/India_Electronic_10F_04_2023_to_12_2023.pdf)
 - [Indonesia: Form DGT-1 CY 2024](/pdfs/Form-DGT-New_Indonesia_FY24_signed.pdf)
 - [Indonesia: Form DGT-1 CY 2023](/pdfs//Form-DGT-New_Indonesia_FY23.pdf)
```

Comment:
Could you remove the double // on line 16?

`[Indonesia: Form DGT-1 CY 2023](/pdfs//Form-DGT-New_Indonesia_FY23.pdf)`
->
`[Indonesia: Form DGT-1 CY 2023](/pdfs/Form-DGT-New_Indonesia_FY23.pdf)`

Reply:
Yes! Great catch :)